src/simulator_core/entities/assets/pump.py

```diff
@@ -65,6 +65,7 @@ def create(self) -> None:
                 in_service=self.in_service,
                 name=self.name,
                 index=self.index,
+                type='pt'
```

Comment:
Doesn't it work without this definition?

Reply:
Ik zal zo eens testen of dit een probleem oplevert wanneer we de pomp op disuse zetten.